runtime/jvmti/jvmtiHelpers.c

```diff
@@ -1929,3 +1929,20 @@ genericWalkStackFramesHelper(J9VMThread *currentThread, J9VMThread *targetThread
 
 	return rc;
 }
+
+#if JAVA_SPEC_VERSION >= 19
+J9VMContinuation *
```

Comment:
I think it would be worth explicitly documenting all of the cases here, since I still don't understand the else case (when would a nonvirtual thread have a continuation?).

Reply:
I guess if the carrier Thread object is used, we would hit this case, but it seems like an error to use the continuation in that case.

Reply:
OK, the case I can see is a carrier thread with a mounted vthread - in that case, the continuation contains the carrier thread stack.

I think this case may need to be checked here to determine whether or not to use the continuation.

Definitely needs documenting!

Reply:
The else case is for a carrier thread. If a vthread is mounted, then the carrier thread details will be stored in J9VMThread->currentContinuation. If a carrier thread is mounted i.e. a vthread is not running, then J9VMThread->currentContinuation will be NULL.